sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md

```diff
@@ -278,6 +278,20 @@ private static class ClientPrivateKeyStrategy implements PrivateKeyStrategy {
 }
 ```
 
+
+### Enable local JRE key store
+
+The local JRE key store can also work with Key Vault key store. To enable this, add the following configuration:
```

Comment:
I think we could give more clear description that it indeed is used as a trust store. Please introduce about the context as well, for example, which is used by default, and what will happen when the below property is enabled. Thanks.